packages/relay/src/lib/precheck.ts

```diff
@@ -205,58 +177,14 @@ export class Precheck {
   /**
    * Checks the balance of the sender account.
    * @param tx - The transaction.
-   * @param account - The account information.
+   * @param accountBalance - The account balance in tinybars.
    */
-  balance(tx: Transaction, account: any): void {
-    const result = {
-      passes: false,
-      error: predefined.INSUFFICIENT_ACCOUNT_BALANCE,
-    };
-
+  balance(tx: Transaction, accountBalance: number): void {
```

Comment:
I saw some tests converting to `number` probably just to satisfy this `accountBalance`'s type. Maybe better if we use

```suggestion
  balance(tx: Transaction, accountBalance: number | string): void {
```

to allow both. And it also it's safer to use `string` than `number`.